packages/ipa/env/ipa/start.coffee

```diff
@@ -1,15 +1,16 @@
 
 path = require 'path'
-os = require 'os'
-nikita = require '@nikitajs/core'
-require '@nikitajs/lxd/lib/register'
-require '@nikitajs/tools/lib/register'
+nikita = require '@nikitajs/engine/src'
+require '@nikitajs/lxd/src/register'
+require '@nikitajs/tools/src/register'
 
 # Note:
 # Jan 20th, 2020: upgrading ubuntu to 19.10
 # lead to an error while installing freeipa
 # complaining that it cannot write into /tmp
 # solution involve `echo '0' > /proc/sys/fs/protected_regular && sysctl -p`
+# Dec 4th, 2020: same for centos/7 20201203_07:08
+# solution involve `chmod -R 777 /tmp`
```

Comment:
Why this has to be recurive with the `-R` flag, provide the link of your source

Reply:
I've been trying following commands:
```
chmod 777 /tmp
chmod 1777 /tmp
chmod -R 1777 /tmp
chmod -R 755 /tmp
```

Only `chmod -R 777 /tmp` installs freeipa without errors.